pyproject.toml

```diff
@@ -28,3 +28,6 @@ requires = [
 
     "scipy>=0.19.1",
 ]
+
+[tool.black]
+line-length = 79
```

Comment:
Not related to this PR. Could you please remove? Fortunately, we will have black with another PR.